pkg/model/loader.go

```diff
@@ -114,9 +114,9 @@ func (ml *ModelLoader) ListModels() []Model {
 	return models
 }
 
-func (ml *ModelLoader) LoadModel(modelName string, loader func(string, string) (*Model, error)) (*Model, error) {
+func (ml *ModelLoader) LoadModel(modelID, modelName string, loader func(string, string, string) (*Model, error)) (*Model, error) {
```

Comment:
sadly.. yes for now :( - some of the models e.g. vLLM can just have a modelName e.g. `owner/repository` which refers to a remote hf repository (and not being local at all).

This code requires a bit of more refactoring here, I'd like to move the model details to its proper structure instead of passing by strings, but I'll keep that for later as would need a bit of more tests to be added first